content/partners/media/kube_careers.md

```diff
@@ -0,0 +1,10 @@
+---
+key: kube_careers
+title: Kube Careers
+category: media
+logo: /images/partners/media/logo-color@2x.png
+website: 'https://kube.careers/'
+socials: []
+---
+
+Hand-picked Kubernetes jobs. Clear salary ranges and apply directly to companies.
```

Comment:
I think we can use the long description version from their media kit, that could be more descriptive.

Reply:
Nice catch, I did not notify there is `description.md` file.